ros2interface/ros2interface/verb/list.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2interface.api import get_all_action_types
+from ros2interface.api import get_all_message_types
+from ros2interface.api import get_all_service_types
+from ros2interface.verb import VerbExtension
+
+
+def print_messages():
+    print('Messages: ')
+    message_types = get_all_message_types()
+    for package_name in sorted(message_types):
+        for message_name in sorted(message_types[package_name]):
+            print('    {package_name}/msg/{message_name}'.format_map(locals()))
+
+
+def print_services():
+    print('Services: ')
+    service_types = get_all_service_types()
+    for package_name in sorted(service_types):
+        for service_name in sorted(service_types[package_name]):
+            print('{package_name}/srv/{service_name}'.format_map(locals()))
```

Comment:
Why are services and actions not indented but messages are?

Reply:
This was resolved in a follow-up PR

Reply:
> This was resolved in a follow-up PR

Please reference the follow-up PR / the change within it in case it is bigger.

Reply:
https://github.com/ros2/ros2cli/pull/289